### PR TITLE
add hyperlinks and small fixes

### PR DESCRIPTION
**Pull request by amnona**
fixed all comments from PR #67 
also added more links and important options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,79 @@ Deblur
...
 ```
-deblur workflow --seqs-fp all_samples.fna --ref-fp db.fna --output-dir output
+split_libraries_fastq.py -i XXX_R1_001.fastq -m map.txt -o split -b XXXX_I1_001.fastq -q 19
 ```
+
+and use the split/seqs.fna as the input to the deblur workflow.
+
+Important options
+=================
+- the sequence read length can be specified by the ```-t NNN``` flag, where NNN denotes the length all sequences will be trimmed to (default=150). Note that all reads shorter than this length will be discarded.
```
Comment (mortonjt) [inline README.md]: the - The

**Comment by mortonjt** [comment]
👍

File: README.md [hunk 2]
```diff
@@ -4,30 +4,79 @@ Deblur
...
+and activate it using:
 
 ```
+source activate debenv
+```
+
+- Install Deblur dependencies:
+```
 conda install -c biocore VSEARCH MAFFT SortMeRNA numpy
-python setup.py install
```
Comment (mortonjt) [inline README.md]: Can you add
```
conda install -c bioconda biom-format 
```

File: README.md [hunk 3]
```diff
@@ -4,32 +4,72 @@ Deblur
...
-=====================
-
-Some of the code in the package deblur has been derived from [QIIME](http://qiime.org).
-The contributors to these specific QIIME modules have granted permission
-for this porting to take place and put under the BSD license.
+Deblur is a greedy deconvolution algorithm based on Illumina Miseq/Hiseq error profiles.
 
 Install
 =======
+- Deblur requires Python 3.5. If Python 3.5 is not installed, you can create a conda environment for deblur using:
```
Comment (wasade) [inline README.md]: The below doesn't create the environment but installs packages into the current environment, so the sentence is inaccurate

**Comment by wasade** [comment]
Would it be possible to add a note about how to ask for help? Not sure if we should setup a google group for this or centralize on the repo

**Comment by mortonjt** [comment]
I think it is ok to refer to people to the issue tracker for now.  What do you think?

**Comment by mortonjt** [comment]
I think this is good to go

File: README.md (hunk 3 above)
Comment (amnona) [inline README.md]: oops, disappeared in the conflict merge...
will fix.

**Pull request merged**